application-services/custom/camera-management/appcamera/types.go

```diff
@@ -0,0 +1,201 @@
+//
+// Copyright (C) 2022 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package appcamera
+
+type StreamUriRequest struct {
+	StreamSetup  StreamSetup `json:"StreamSetup"`
+	ProfileToken string      `json:"ProfileToken"`
+}
+type Transport struct {
+	Protocol string `json:"Protocol"`
+}
+type StreamSetup struct {
+	Stream    string    `json:"Stream"`
+	Transport Transport `json:"Transport"`
+}
```

Comment:
Why have json attributes that are the default values used without having the attributes. Typically the json uses camel case. Same for rest of json attributes.
```suggestion
type StreamUriRequest struct {
	StreamSetup  StreamSetup `json:"streamSetup"`
	ProfileToken string      `json:"profileToken"`
}
type Transport struct {
	Protocol string `json:"protocol"`
}
type StreamSetup struct {
	Stream    string    `json:"stream"`
	Transport Transport `json:"transport"`
}
```

Reply:
This code was generated from an input json string, and the tags were added automatically. I believe the actual json payload needs to be capitalized. In this case do you suggest I remove the json tags altogether or just keep as is?

Reply:
Interesting... the json tags are redundant, but since this is generated code I would just add comment that it is generated code and it is what it is... ;-)